cli/command/engine/check.go

```diff
@@ -0,0 +1,123 @@
+package engine
+
+import (
+	"fmt"
+
+	"golang.org/x/net/context"
```

Comment:
it's just "context" now

Reply:
Thanks - I'll clean this up - I started this work quite a while back and I think the CLI was still using that pattern when I started... just overlooked switching these as I rebased.